main/core.js

```diff
@@ -42,6 +43,29 @@ async function setup (ctx) {
       await fs.writeFile(filePath, logs.get())
     }
   }
+
+  ctx.toggleOpenAtLogin = () => {
```

Comment:
The `main/core.js` file is about managing the Station Core instance, not the "core" app functions. What do you think about creating `main/settings.js`, and managing these `ctx` methods there? It would also have a public `setup` method, which would be run in `main/index.js`